test/CMakeLists.txt

```diff
@@ -1056,3 +1056,8 @@ endif()
 if(MIOPEN_TEST_FLOAT)
     add_custom_test(test_reduce_double SKIP_UNLESS_ALL GFX908_ENABLED COMMAND  $<TARGET_FILE:test_reduce_test> --double --all --verbose)
 endif()
+
+# REGRESSION TEST for issue #894
+add_custom_test(test_miopen_conv_ocl_half_regression SKIP_UNLESS_ALL FLOAT_DISABLED HALF_ENABLED GFX908_ENABLED 
+COMMAND	MIOPEN_FIND_MODE=normal MIOPEN_DEBUG_FIND_ONLY_SOLVER=ConvOclDirectFwd1x1 $<TARGET_FILE:test_conv2d>  --verbose   --half  --cmode conv    --pmode default --group-count   1 --input	1   16	7	7	--weights	16	16	1	1	--pads_strides_dilations	0	0	1	1	1	1
```

Comment:
Please add `--disable-backward-data --disable-backward-weights` or this will always fail. I guess it's also worth adding `--disable-verification-cache`.